usr/src/test/smartos-test/README

```diff
@@ -78,6 +78,12 @@ or
     # /opt/os-tests/bin/ostest
 
 
+Notes on NVMe testing:
+
+NVMe is tested by unit tests and by non-destructive component tests.  The
+latter must specify NVME_TEST_DEVICE from nvme* names.  E.g. "nvme0".  If
+NVME_TEST_DEVICE is not run, the non-destructive component tests will not run.
```

Comment:
> If the NVME_TEST_DEVICE is not run, ...

Should that be "If NVME_TEST_DEVICE is not *set*"?

Reply:
Correct.  Will fix!